src/users/users.module.ts

```diff
@@ -2,7 +2,7 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 import { Module } from '@nestjs/common';
 import { UsersController } from './controller/users/users.controller';
 import { UsersService } from './service/users/users.service';
-import { User } from 'src/typeorm/User';
+import { User } from 'src/typeorm/entity/user.entity';
```

Comment:
```suggestion
import { User } from 'src/typeorm/entities/user.entity';
```

Tên folders đặt số nhiều, trừ module